components/script/dom/eventdispatcher.rs

```diff
@@ -103,10 +103,15 @@ fn dispatch_to_listeners(event: &Event, target: &EventTarget, event_path: &[&Eve
     }
 }
 
+pub enum EventStatus {
```

Comment:
I actually like `if let`, but if you think it is necessary I could change the code to use `==`